src/main/java/core/basesyntax/service/operation/BalanceOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public BalanceOperationHandler(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction, StorageService storageService) {
```

Comment:
According to the checklist, the `handle` method should not take `StorageService` as a parameter since the `OperationHandler` should be independent of the services.